src/w3d/renderer/motchan.cpp

```diff
@@ -196,14 +196,14 @@ bool TimeCodedMotionChannelClass::Load_W3D(ChunkLoadClass &cload)
         return false;
     }
 
-    m_numTimeCodes = chan.NumTimeCodes;
-    m_vectorLen = chan.VectorLen;
     m_type = chan.Flags;
+    m_vectorLen = chan.VectorLen;
+    m_numTimeCodes = chan.NumTimeCodes;
+    m_packetSize = chan.VectorLen + 1;
+    m_lastTimeCodeIdx = (chan.VectorLen + 1) * (m_numTimeCodes - 1);
```

Comment:
can remain m_packetSize * (m_numTimeCodes - 1)

Reply:
Yes. But I think writing it this way is much cleaner, because this way it only uses the chan struct to set class members, and not other class members to set class members, thus removing any kind of order dependencies.

Compiler can optimize it any way it wants, and it will likely make it one `chan.VectorLen + 1` only.